boards/arm/ubx_bmd330eval_nrf52810/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+# BMD-330-EVAL board configuration
+
+# Copyright (c) 2021 u-blox AG
+# SPDX-License-Identifier: Apache-2.0
+
+# The ubx_bmd330eval_nrf52810 has the same pin-out as the pca10040,
+# though it uses the nrf52810 directly. This needs to be considered
+# by certain system initialization functionality residing in
+# system_nrf52810.c and SoC dependent routines in nrfx_coredep.h.
+
+# zephyr_compile_definitions(DEVELOP_IN_NRF52832) = not needed. the
+# BMD-330 uses the nrf52810, so nothing is emulated.
+
+# The following line is required to correct timing for nrf_delay_us
+zephyr_compile_definitions(NRFX_COREDEP_DELAY_US_LOOP_CYCLES=3)
```

Comment:
Since the board uses the actual nRF52810, this value should not be overridden here (one loop iteration will in fact take 7 cycles).
The whole file is actually unnecessary and I think it would be better to remove it.

Reply:
Thanks - I'll remove the file in the next push. I re-read the Infocenter docs and saw my error.